project/index.js

```diff
@@ -7,6 +7,52 @@ const errHelper = (err, res) => {
   res.status(500).json({ message: `internal error server ${err}` });
 };
 
+//  [GET] all projects with all actions and paginations
+//  [QUERY] by name
+server.get("/", async (req, res) => {
+  const { limit = 10, page = 1, name } = req.query;
+
+  try {
+    let projects;
+    if (!name) {
+      projects = await db
```

Comment:
Try to find ways to shorten your db queries to reduce the flatness of your backend